toolchain/sem_ir/stringify_type.cpp

```diff
@@ -177,6 +177,11 @@ auto StringifyTypeExpr(const SemIR::File& outer_sem_ir, InstId outer_inst_id)
         }
         break;
       }
+      case CARBON_KIND(FacetAccessType inst): {
+        // Print `(T as I) as type` as simply `T`.
```

Comment:
I think this will print it as `T as I`, if it's a `FacetValue`. (Which it generally won't be because we'll have evaluated that to simply `T` already). Maybe:
```suggestion
        // Given `T:! I`, print `T as type` as simply `T`.
```